classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.Classroom = self
+  end
+end
```

Comment:
- [ ] I am making this comment here, but the comment is intended for your student.rb class, but it is not part of the file you worked on, so I have to make the comment here, in your student.rb class, there should be a  `method that serves as the 'belongs-to' side (a student belongs to a classroom)`.this method should look like this
```
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```
and classroom should be made accessible with attr_accessor, I hope this is clear enough